weaviate/collection/config.py

```diff
@@ -27,11 +27,14 @@ class _ConfigBase:
     """
 
     def __init__(self, connection: Connection, name: str) -> None:
-        self.__cached: Optional[Dict[str, Any]] = None
         self.__connection = connection
         self._name = name
 
-    def _fetch(self) -> Dict[str, Any]:
+    @classmethod
+    def make(cls, connection: Connection, name: str):
```

Comment:
why is this better?

Reply:
Ah, that's a relic from the strict/debug/config PR that this branched from. Will remove for clarity